astropy/units/required_by_vounit.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This package defines SI prefix ed units that are required by the VOUnit standard
```

Comment:
Actually, the text reads a bit oddly. How about the following to replace the first
two paragraphs:
```
This package defines SI prefixed units that are required by the VOUnit standard
but that are rarely used in practice and liable to lead to confusion (such as
``msolMass`` for milli-solar mass). They are in a separate module from
``astropy.units.deprecated`` because they need to be enabled by default for
``astropy.units`` to parse compliant VOUnit strings. As a result, e.g.,
``Unit('msolMass')`` will just work, but to access the unit directly one has to
use ``astropy.units.required_by_vounit.msolMass`` instead of the more
usual route possible for the non-prefixed unit, ``astropy.units.solMass``.
```